scripts/upgrade

```diff
@@ -40,6 +40,10 @@ fi
 
 ynh_backup_before_upgrade	# Backup the current version of the app
 ynh_clean_setup () {
+	# Delete any created temporary folder
+	if [ -v tmpdir ] && [ -d $tmpdir ]; then
```

Comment:
What is `-v` ?

Reply:
```
-v varname

    True if the shell variable varname is set (has been assigned a value).
```
as explained [here](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html) (requires Bash 4.2+)

Reply:
Ok, that's the problem between `test` and `[`, the man aren't the same. But this option works for both...
Hopefully it's used after `set +eu` !